changelog/unreleased/6540.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "type": "Bug fixes and improvements",
```

Comment:
I see how we can easily make a typo here and get a weird header. Can we make it fool-proof? For example:
We only have a finite number of headers. Features, Bug fixes and improvements, Known issues.
Can we create directories for them? And then what's left is place a change entry as a plain string.
So for this example it would be:
1) find directory "changelog/unreleased/Bug fixes and improvements"
2) Create a file "6540.json"
3) The file contents:
```
[
"Fixed approaches list update in `RouteOptionsUpdater`(uses for reroute). It was putting to the origin approach corresponding approach from legacy approach list."
]
```
If there are multiple entries, we just add a second array item.

What I want to do here is to not use any json keys. Because it creates room for errors as well as typing "Bug fixes and improvements" every time.
I think an array of string would be sufficient, WDYT?

Reply:
If the only field that left in file is title, maybe we can replace `json` by `md`?

Reply:
I think it's a good point. In our case we have only 2 main parts: features and bugfixes, we can move type to path and make changelog file as md file